proxy/src/main/java/org/apache/rocketmq/proxy/grpc/interceptor/AuthenticationInterceptor.java

```diff
@@ -48,7 +48,7 @@ public <R, W> ServerCall.Listener<R> interceptCall(ServerCall<R, W> call, Metada
             @Override
             public void onMessage(R message) {
                 GeneratedMessageV3 messageV3 = (GeneratedMessageV3) message;
-                headers.put(InterceptorConstants.RPC_NAME, messageV3.getDescriptorForType().getFullName());
+                headers.put(InterceptorConstants.RPC_NAME, messageV3.getDescriptorForType().getName());
```

Comment:
FullName is needed for future version compatibility.

Reply:
I have followed your suggestion and used the new SIMPLE_RPC_NAME attribute to record the action name needed later.

Reply:
@drpmma Please take a look.